src/dmd/frontend.h

```diff
@@ -3562,7 +3562,7 @@ class ToCppBuffer final : public Visitor
     void visit(AliasDeclaration* ad);
     void visit(Nspace* ns);
     void visit(CPPNamespaceDeclaration* ns);
-    void handleNspace(Dsymbol* namespace, Array<Dsymbol* >* members);
+    void handleNspace(Dsymbol* namespace_, Array<Dsymbol* >* members);
```

Comment:
Could just not emit parameter names?  They are just superfluous information at best.

Reply:
Sanitizing here as well makes sense IMO, especially as all comments are stripped, so documentation is scarce already.

Reply:
Hmmm, maybe it would be useful if we could generate the documentation for the C++ headers also?